ChartsDemo-iOS/Swift/Demos/PiePolylineChartViewController.swift

```diff
@@ -6,9 +6,7 @@
 //  Copyright © 2017 jc. All rights reserved.
 //
 
-#if canImport(UIKit)
```

Comment:
why we change here? I remember it's for SPM recently

Reply:
It wasn't ever necessary. SPM doesn't compile `ChartsDemo-iOS/`